site/en/blog/100-web-moments/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Celebrate Chrome 100 with #100CoolWebMoments'
+description: >
+  In 
+layout:
+  'layouts/blog-post.njk'
+date: 2022-03-29
+authors:
+  - rachelandrew
+hero: 'image/kheDArv5csY6rvQUJDbWRscckLr1/AlyVvShYHBM5I7ikG8vj.jpg'
+alt: >
+  100 cool web moments.
+tags:
+  - chrome-100
+---
+
+Google Chrome launched in 2018, and in March 2022 we are celebrating the launch of Chrome version 100 with a walk down memory lane. [#100CoolWebMoments](/100) remembers some cool, fun, interesting, and important things that have happened in the past 14 years.
+
+A lot has happened on the web since 2018. If you read Ethan Marcotte’s 2010 A List Apart article on [responsive web design](https://alistapart.com/article/responsive-web-design/) on the day of publication you might have done so with Chrome 4. By the time [Media Queries became a W3C Recommendation](https://www.css3.info/css3-media-queries-becomes-official-w3c-recommendation-css4-media-queries-takes-shape/) in 2012, Chrome 19 was available.
+
+Chrome was at version 43 when Alex Russell introduced [Progressive Web Apps](https://infrequently.org/2015/06/progressive-apps-escaping-tabs-without-losing-our-soul/) in 2015, with [Microsoft announcing Edge as the new browser](https://www.theverge.com/2015/4/29/8511169/microsoft-edge-official-name-internet-explorer-upgrade) to replace Internet Explorer in the same year. 
+
+In March 2017, version 57 brought us CSS Grid, the layout method was shipped in [Chrome, Firefox, and Safari](https://rachelandrew.co.uk/archives/2017/03/07/css-grid-lands-in-firefox-52/) within days of each other. This shared release of a major feature happened again more recently, with [Cascade Layers](/blog/cascade-layers/) landing in Chrome 99, along with Firefox 97, and Safari 15.4. 
+
+Tell us which moments you loved the most. If we've missed anything (and we're sure we have), tweet us [@Chromiumdev](https://twitter.com/ChromiumDev) with [#100CoolWebMoments](https://twitter.com/hashtag/100CoolWebMoments). 
```

Comment:
Nit: the `@ChromiumDev` capitalization is easier to read than `@Chromiumdev`, and matches the actual Twitter display name for the handle.